signer-wasm/src/ledger.rs

```diff
@@ -11,16 +11,40 @@ use bip44::BIP44Path;
 
 use crate::utils::{address_to_object, bytes_to_buffer, signature_to_object, Buffer};
 
+const INVALID_BIP44: &str = "Invalid BIP44 Path";
+
 macro_rules! ok_or_ret_promise {
```

Comment:
should this be renamed `ok_or_reject_promise` or ``ok_or_reject` ? I think it would make the rejection a bit more explicit in the places where the macro is used

Reply:
Indeed, `ok_or_reject` makes a stronger intention